.github/config/en-custom.txt

```diff
@@ -686,4 +686,33 @@ roundtrips
 reimplement
 keyspace
 upsert
-TTL
\ No newline at end of file
+TTL
+IRSA
+Entra
+radified
+OpenID
+multitenancy
+roleARN
+AWSCredentialProvider
+AssumeRole
+AssumeRoleProvider
+STS
+assumeRoleProviders
+stscreds
+AWSCredential
+AWSCredentialProvider
+ctx
+func
+newAWSConfig
+nithya
+planeName
+POC
+TokenExpirationSeconds
+amazonaws
+serviceaccount
+oidc
+assumeRoleProvider
+TokenExpirationSeconds
+amazonaws
+serviceaccount
```

Comment:
No need to go back and change it, but for future reference....

The spellcheck will ignore anything in a code block. If you're seeing a bunch of technical terms like `assumeRoleProvider` in the spellcheck warnings, chances are you missed a code block for something that is code 👍